backend/src/hyde_config.rs

```diff
@@ -0,0 +1,58 @@
+use std::fs;
```

Comment:
```suggestion
use fs_err as fs;
```
We already pull in `fs_err` for pretty errors, might as well use it here

Reply:
I don't see this used anywhere in the source?